cvat-canvas3d/src/typescript/canvas3dModel.ts

```diff
@@ -398,6 +410,10 @@ export class Canvas3dModelImpl extends MasterImpl implements Canvas3dModel {
             this.data.shapeProperties.outlineColor = shapeProperties.outlineColor;
         }
 
+        if (typeof shapeProperties.showAxisArrows === 'object') {
+            this.data.shapeProperties.showAxisArrows = shapeProperties.showAxisArrows;
```

Comment:
A little safer from types pov:

```suggestion
        if (typeof shapeProperties.showAxisArrows === 'object') {
            const current = this.data.shapeProperties.showAxisArrows;
            current.x = !!(shapeProperties.showAxisArrows?.x ?? current.x);
            current.y = !!(shapeProperties.showAxisArrows?.y ?? current.y);
            current.z = !!(shapeProperties.showAxisArrows?.z ?? current.z);
```

Reply:
Or something like this
Idea is not just check that input type is object